src/style.css

```
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.form-big-small {
+  position: relative;
+  margin: 20px 0;
+}
+
+.search-input {
+  width: 100%;
+  box-sizing: border-box;
```

```suggestion
  box-sizing: border-box;
```

You can add `box-sizing` to all tags 
```
* {
  box-sizing: border-box;
  }
```